internal/network/command_create.go

```diff
@@ -0,0 +1,169 @@
+package network
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: "Create a network.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.create,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create a Confluent network in AWS with connection type "transitgateway" by specifying zones and CIDR.`,
+				Code: "confluent network create aws_tgw --cloud aws --region us-west-2 --connection-types transitgateway --zones usw2-az1,usw2-az2,usw2-az4 --cidr 10.1.0.0/16",
+			},
+			examples.Example{
+				Text: `Create a Confluent network in AWS with connection type "peering" by specifying zones info.`,
+				Code: "confluent network create aws_peering --cloud aws --region us-west-2 --connection-types peering --zone-info usw2-az1=10.10.0.0/27,usw2-az3=10.10.0.32/27,usw2-az4=10.10.0.64/27",
+			},
+			examples.Example{
+				Text: `Create a Confluent network in GCP with connection type "peering" by specifying zones and CIDR.`,
+				Code: "confluent network create gcp_peering --cloud gcp --region us-central1 --connection-types peering --zones us-central1-a,us-central1-b,us-central1-c --cidr 10.1.0.0/16",
+			},
+			examples.Example{
+				Text: `Create a Confluent network in Azure with connection type "privatelink" by specifying DNS resolution.`,
+				Code: "confluent network create azure_pl --cloud azure --region eastus2 --connection-types privatelink --dns-resolution chased_private",
```

Comment:
Feel free to replace all "_" with "-" for values passed to `--dns-resolution`. We should accept both "chased_private" and "chased-private" but the latter should be preferred since they match the style of the CLI better.
```suggestion
				Code: "confluent network create azure-pl --cloud azure --region eastus2 --connection-types privatelink --dns-resolution chased-private",
```